lib/survey_gizmo/rest_response.rb

```diff
@@ -48,12 +48,20 @@ def initialize(rest_response)
 
   # The parsed JSON data of the response
   def data
-    @_data ||= @parsed_response['data']
+    @parsed_response['data']
   end
 
   # The error message if there is one
   def message
-    @_message ||= @parsed_response['message']
+    @parsed_response['message']
```

Comment:
(i think in the old code base there was significant reuse and manipulation of `@_data` but that has long since been removed)